web/frontend/package.json

```diff
@@ -14,21 +14,21 @@
     }
   },
   "peerDependencies": {
-    "@improbable-eng/grpc-web": "~0.15.*",
-    "@viamrobotics/prime": "~0.2.*",
-    "@viamrobotics/rpc": "~0.1.*",
+    "@improbable-eng/grpc-web": ">=0.15",
```

Comment:
Improves peer-dep resolutions. @mcous called this out a while ago, I am finally getting around to fixing it!